nginx-nodejs-redis/web/server.js

```diff
@@ -0,0 +1,24 @@
+cost os = require('os');
```

Comment:
I detected a compilation error when testing the sample from scratch
```suggestion
const os = require('os');
```